package.json

```diff
@@ -62,6 +62,7 @@
     "babel-plugin-macros": "^3.0.1",
     "babel-plugin-module-resolver": "^4.1.0",
     "chalk": "^4.1.0",
+    "commitizen": "^4.2.3",
```

Comment:
I'm not keen on bundling commitizen into skuba itself while it has [high-severity security vulnerabilities](https://snyk.io/test/npm/commitizen/4.2.3) that have gone unpatched for months. If we're sure this is still the best tool for the job (what does it do besides present ~2 prompts to the user?) I'd place it in the template `package.json`.

Reply:
Cool, can move it to templates then 👍

Reply:
I guess sec is less of a concern for dev deps on packages